chia/server/upnp.py

```diff
@@ -29,7 +29,10 @@ def run():
                     if msg[0] == "remap":
                         port = msg[1]
                         log.info(f"Attempting to enable UPnP (open up port {port})")
-                        self.upnp.deleteportmapping(port, "TCP")
+                        try: 
+                            self.upnp.deleteportmapping(port, "TCP")
+                        except : 
```

Comment:
LGTM doesn't like the blanket except, maybe catch just the expected error here

Reply:
Thanks btw!!!

Reply:
I think it would make sense to log the exception too